### DISCUSSION TITLE
Windows 11

### DISCUSSION THREAD

**Discussion by Panja0** [farag2/Sophia-Script-for-Windows#324] [Q&A] [answered]
Is Sophia script compatible with Windows 11 build: 22000.318?

Because it’s noted that the minimum is 376.

**Comment by farag2** [comment, marked as answer]
Physically yes. But I checked the current latest build number released by MS. So what's the problem to run the WU snd update the OS?

**Comment by Panja0** [comment]
I would like to run it on a clean install. 
Latest installable/downloadable official iso is 318. 

You think it’s better to install and than run win update to update to latest build? Win update Will install a lot more updates (drivers firmware etc).

**Reply by farag2** [reply]
You're god damn right. Run the WU. Why is it bad to update the OS, its' components and drivers?

**Comment by Panja0** [comment]
LOL!
Not at all bad! But I’ve read the following:
_Running the script is best done on a fresh install because running it on wrong tweaked system may result in errors occurring._

So I thought it would be better to run Sophia before I updated.

**Reply by farag2** [reply]
This was just a warning for users who like to make harm to their OSes by running a random scripts like from Sycnex's.

**Reply by Panja0** [reply]
Understood! Thanks for clearing that up.

**Reply by farag2** [reply]
You're welcome )
